src/utils/app.utils.ts

```diff
@@ -0,0 +1,68 @@
+import { EventName, EventStatus, StatusMessages } from 'src/common/constants';
+import { FormattedData } from 'src/common/types/webhook.event.types';
+
+export function isPayloadValid(payload: any): boolean {
+  return (
+    payload?.action &&
+    payload?.pull_request &&
+    payload?.repository &&
+    payload?.sender &&
+    payload?.installation
+  );
+}
+
+export function createFormattedData(
+  partialData: Partial<FormattedData>,
+): FormattedData {
```

Comment:
PullProofAI

Consider adding type annotations for the `createFormattedData` function parameters for better type safety.
```diff
+   partialData: Partial<FormattedData>
```


<!-- This is an auto-generated comment by PullProofAI -->